manifests/config/passenger.pp

```diff
@@ -95,6 +95,13 @@
   include ::apache::mod::headers
   include ::apache::mod::passenger
 
+  if $::foreman::ipa_authentication {
```

Comment:
This should be added to the local class parameters list so it can be set in foreman::config::passenger tests.